src/scalap/scala/tools/scalap/scalax/rules/scalasig/ScalaSig.scala

```diff
@@ -15,8 +15,8 @@ package scalax
 package rules
 package scalasig
 
-import scala.language.postfixOps
 import scala.language.implicitConversions
+import scala.language.postfixOps
```

Comment:
Did you mean to remove this one, or just to swap its place?

Reply:
It is required. My modus operandi was to remove the import and observe what broke. In this case, I restored the import in sorted order. Soon, the build will have `-Xlint` enabled and we won't have to guess about unused imports.